less/navigationBar.less

```diff
@@ -546,394 +456,27 @@
-  }
-
-  .urlbarForm {
-    .loadTime {
-      color: @loadTimeColor;
-      // background: @navigationBarBackground;
-      font-size: 12px;
-      text-align: right;
-      cursor: default;
-
-      &.onFocus {
-        background: @navigationBarBackgroundActive;
-        display: none;
-      }
-    }
-
-    /* Disable selection of button text */
-    > span {
-       user-select: none;
-    }
-
-    .inputbar-wrapper {
-      //background: white;
-      display: flex;
-      flex: 1 1 0;
-      border-radius: 4px;
-      align-items: center;
-      justify-content: center;
-    }
-
-    input {
-      background: @navigationBarBackgroundActive;
-      border: none;
-      box-sizing: border-box;
-      color: #333;
-      letter-spacing: -0.125px;
-      cursor: text;
-      flex-grow: 1;
-      font-size: 13.5px;
-      font-weight: normal;
-      outline: none;
-      text-overflow: ellipsis;
-      min-width: 0%; // allow the navigator to shrink
-
-      &:hover {
-        background: @navigationBarBackgroundActive;
-      }
-
-      &.private {
-        background: @privateTabBackground;
-        color: @chromeText;
-      }
-
-      &:focus {
-        background: @navigationBarBackgroundActive;
-        margin-right: 3px;
-      }
-    }
-
-    // Create 25x25 square and place .urlbarIcon at the center of it
-    .urlbarIconContainer {
-      display: flex;
-      align-items: center;
-      justify-content: center;
-      height: @urlbarFormHeight;
-      width: @urlbarFormHeight;
-      min-height: @urlbarFormHeight;
-      min-width: @urlbarFormHeight;
-
-      .urlbarIcon {
-        color: @siteSecureColor;
-        font-size: 12px;
-        background-repeat: no-repeat;
-        background-position: center;
-        position: relative;
-        bottom: -1px;
-
-        // about:newtab
-        &.fa-search {
-          position: relative;
-          bottom: 0;
-          // 50% of #5a5a5a
-          color: rgba(90, 90, 90, 0.5);
-        }
-
-        &.fa-lock,
-        &.fa-unlock,
-        &.fa-exclamation-triangle {
-          font-size: 16px;
-          bottom: 0;
-        }
-
-        &.insecure-color {
-          color: @siteInsecureColor;
-        }
-
-        &.extendedValidation {
-          color: @siteEVColor;
-        }
-      }
-    }
-  }
-}
-
-.topLevelEndButtons .braveMenu {
-  background-image: -webkit-image-set(url(../app/extensions/brave/img/braveBtn.png) 1x,
-                                      url(../app/extensions/brave/img/braveBtn2x.png) 2x,
-                                      url(../app/extensions/brave/img/braveBtn3x.png) 3x);
-  background-repeat: no-repeat;
-  height: 24px;
-
-  &:hover {
-    background-image: -webkit-image-set(url(../app/extensions/brave/img/braveBtn_hover.png) 1x,
-                                        url(../app/extensions/brave/img/braveBtn2x_hover.png) 2x,
-                                        url(../app/extensions/brave/img/braveBtn3x_hover.png) 3x);
```

Comment:
@cezaraugusto do you see that the icon image slightly disappears on hover? I do, on my packaged build. It would be resolved if there is a way of preloading those image.